.ebextensions/options.config

```diff
@@ -17,7 +17,8 @@ option_settings:
     FLASK_ENV: production
 
   aws:elasticbeanstalk:container:python:
-    WSGIPath: application.py
+    WSGIPath: application:application
 
-  aws:elasticbeanstalk:container:python:staticfiles:
-    /static/: deepcell_label/static/
+packages: 
+  yum:
+    mariadb-devel: []
```

Comment:
No longer necessary to provide the staticfiles path?

Reply:
The static files are now hosted in an S3 bucket instead of served by the flask backend